css-two-column-layout/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  font-family: "Karla", sans-serif;
+  line-height: 1.75rem;
+}
+
+h1 {
+  font-size: 40pt;
+  font-weight: bolder;
+}
+
+img {
+  /* height: 100%; */
+}
```

Comment:
This can be removed. Make sure to remove all commented code prior to submitting your work